rxbinding/src/main/java/com/jakewharton/rxbinding/widget/SearchViewQueryTextEvent.java

```diff
@@ -0,0 +1,52 @@
+package com.jakewharton.rxbinding.widget;
+
+import android.support.annotation.NonNull;
+import android.widget.SearchView;
+
+import com.jakewharton.rxbinding.view.ViewEvent;
+
+public class SearchViewQueryTextEvent extends ViewEvent<SearchView> {
+    private final CharSequence queryText;
+    private final boolean submitted;
+
+    protected SearchViewQueryTextEvent(@NonNull SearchView view,
```

Comment:
should be private. creation is done with a static factory method which needs added.